app.js

```diff
@@ -1,14 +1,34 @@
+'use strict';
 
+/* External modules */
 
-var express = require('express'),
-  config = require('./config/config'),
-  glob = require('glob');
+var fs = require('fs');
+var express = require('express');
+var winston = require('winston');
+
+/* Library modules */
+
+var CampaignInfoStorage = require('./lib/storage/sqlite/campaign-info-storage');
+
+/* Controllers */
+
+var campaigninfoById = require('./controllers/campaigns').campaigninfoById;
+
+/* App variables */
 
 var app = express();
+var config = JSON.parse(fs.readFileSync('config.json'));
+var extern = { logger: winston };
 
```

Comment:
Hmmm, I completely agree. However, if we implement reverse proxy or similar, will that display the headers from `nginx` or `node`.

Disclaimer: I am not completely sure how this part works w.r.t nginx/node, some expert advise is needed.